README.md

```diff
@@ -582,7 +582,7 @@ var outputPromise = getInputPromise().then(function (input) {
 
 * **Using Async Await**  
 
-Async await makes asynchronous code look like it\’s synchronous. This has only been possible because of the reintroduction of promises into node.js. Async-Await only works with functions that return a promise.
```

Comment:
Escaping apostrophe is required here, becuase it prevent page break while deploying in GitHub Pages.

Reply:
Should I add one to `it's`s in lines 1517 and 1551 then?

Reply:
yep, it should be `it\'s`

Reply:
Changed